core/src/main/java/hudson/model/ManagementLink.java

```diff
@@ -117,7 +117,7 @@ public Permission getRequiredPermission() {
      * Define if the rendered link will use the default GET method or POST.
      * @return true if POST must be used
      * @see RequirePOST
-     * @since TODO
+     * @since 1.584
```

Comment:
I did a search for all `@since TODO` and `@since 1.FIXME` and then found what revision they arrived from